contracts/StMATIC.sol

```diff
@@ -382,17 +382,7 @@ contract StMATIC is
         uint256 operatorInfosLength = operatorInfos.length;
 
         for (uint256 i = 0; i < operatorInfosLength; i++) {
-            IValidatorShare validatorShare = IValidatorShare(
-                operatorInfos[i].validatorShare
-            );
-            uint256 stMaticReward = validatorShare.getLiquidRewards(
-                address(this)
-            );
-            uint256 rewardThreshold = validatorShare.minAmount();
-
-            if (stMaticReward < rewardThreshold) {
-                validatorShare.withdrawRewards();
-            }
```

Comment:
Why is this removed? This prevents distributeRewards to fail if liquid reward is less than minAmount.

Reply:
I moved the logic inside the NodeOperatorContract
https://github.com/Shard-Labs/PoLido/blob/ido-fix/post-audit/contracts/NodeOperatorRegistry.sol#L959